pages/api/addCheckout.ts

```diff
@@ -0,0 +1,71 @@
+import { NextApiHandler } from 'next';
+import { createHash } from 'crypto';
+
+const addCheckout: NextApiHandler = async (req, res) => {
+  const {
+    name,
+    surname,
+    address,
+    phone,
+    deliveryService,
+    paymentType,
+    paymentTypeCode,
+    items = [],
+    totalSum,
+  } = req.body;
+
+  if (
```

Comment:
move check to constant and
const check = name && surname && ... 
if (!check) {
       return res.status(400).send({ message: 'Bad Request' });
}